database/schema.sql

```diff
@@ -0,0 +1,12 @@
+DROP DATABASE IF EXISTS photos;
```

Comment:
smart, I'm actually changing my mongoDB to only update tomorrow.  This seems like the right move.